config

```diff
@@ -2,10 +2,10 @@
 
 IMAGE_NAME="kubevirt/libvirt"
 
-COPR_VERSION="av-8.1"
+COPR_VERSION="av-8.2"
 FEDORA_VERSION="31"
-LIBVIRT_VERSION="5.6.0-7.fc${FEDORA_VERSION}"
-QEMU_VERSION="4.1.0-15.fc${FEDORA_VERSION}"
+LIBVIRT_VERSION="6.0.0-16.fc${FEDORA_VERSION}"
```

Comment:
I think the upstream version should be 6.2. The av-8.2 is based on 6.0 with lots of backports from 6.2, such as virtio-fs.
I would like the upstream to be based on 6.2
@rmohr @stu-gott @davidvossel  what do you think?

Reply:
my vote is to use 6.2 if that more closely matches av-8.2 with the backports.

Reply:
The issue is, there is no 6.2 build available for Fedora (please correct me if I'm wrong).

Reply:
@phoracek afaik, there should be `libvirt  6.2.0-1.fc31` - https://copr.fedorainfracloud.org/coprs/g/virtmaint-sig/virt-preview/packages 
if it's not in the correct repo, we should request to make it available there.

Reply:
https://copr.fedorainfracloud.org/coprs/g/virtmaint-sig/virt-preview/build/1326536/ has 6.2, but it would disappear on the next copr build. I think it is kinda-fine to use it to create our test image, but I think @rmohr dislikes the idea.

Anyway, we should have *something* from the libivrt-6.y series.

Reply:
Another point is that the av-8.2 QEMU matches QEMU_VERSION=4.2.0-7
In any case, even if we will not go with libvirt 6.2 right now, I don't see 6.0 being available in libvirt 31 as well.

Reply:
I think we should use the existing for-kubevirt-av-8.2 content for the purpose of this PR, and work on using fresher Fedora-based content as a follow-up.

Reply:
> https://copr.fedorainfracloud.org/coprs/g/virtmaint-sig/virt-preview/build/1326536/ has 6.2, but it would disappear on the next copr build. I think it is kinda-fine to use it to create our test image, but I think @rmohr dislikes the idea

Indeed. The "test" image is also the "release" image.